test/verify/check-storage-partitions

```diff
@@ -95,6 +95,10 @@ class TestStoragePartitions(StorageCase):
         b.key_press(chr(37), use_ord=True)  # arrow left
         self.dialog_wait_val("size", 25.5)
 
+        # Check that changing units affects the text input
+        b.select_from_dropdown(".size-unit", "1073741824")
```

Comment:
Confused.. The [firefox test fails](https://logs.cockpit-project.org/logs/pull-16046-20210706-094349-c82f3f97-fedora-34-firefox/log.html#284) on it not knowing what `chr(37)` is, but that code already existed!? It's breaking before this new line.

Reply:
Ehm this line you quote here (where is fails) is new and indeed the hardcoded keys for firefox did not include arrow left, reposted.

Reply:
Right, but it failed on line 95 before, no? with the key_press(chr(37))

Reply:
Ah, nevermind -- I only looked at the *second* patch, but the chr(37) is from the first. Thanks for fixing!